pkg/types/azure/defaults/machines.go

```diff
@@ -0,0 +1,20 @@
+package defaults
+
+import (
+	"fmt"
+)
+
+func BootstrapMachineType(region string) string {
```

Comment:
The rest of the functions in the package are name `<>InstanceType`, while this is `BoostrapMachineType`, i think we should keep the `<>InstanceType` for all functions.

Reply:
I've renamed this to BootstrapInstanceType to be consistent with the rest of the functions in this module.